### PR TITLE
Add refund feature spec

### PR DESCRIPTION
**Pull request by nvandoorn**
What is the goal of this PR?
---

Prior to this commit, testing the refund feature manually was cumbersome as we developed fixes for this feature.

### CODE REVIEW

**Review by AlistairNorman — APPROVED** [review]

**Review by benjaminwil — APPROVED** [review]

File: spec/features/spree/admin/refund_spec.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+    click_on "Create reimbursement"
+    select "Original payment", from: "reimbursement[return_items_attributes][0][override_reimbursement_type_id]"
+    click_on "Update"
+
+    perform_enqueued_jobs do
+      click_on "Reimburse"
+    end
+
+    expect(find(".reimbursement-refund-amount")).to have_content("$10.89")
```
Comment (forkata) [inline spec/features/spree/admin/refund_spec.rb:50]: Are we planning on adding any assertions for the transaction being reported to TaxJar or is that going to be done in a different PR?

**Review by forkata — APPROVED** [review]

File: spec/features/spree/admin/refund_spec.rb (hunk 1 above)
Comment (benjaminwil) [reply spec/features/spree/admin/refund_spec.rb:50]: It would be nice to add assertions that verify that the reporting was successful if that is exposed in the admin interface somewhere.
